src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -234,12 +240,6 @@ private void InitializeCore()
             ChangeRootCategoryExpandState(BuiltinNodeCategories.GEOMETRY, true);
         }
 
-        private void LibraryRootOnPropertyChanged(object sender, PropertyChangedEventArgs propertyChangedEventArgs)
-        {
-            if (propertyChangedEventArgs.PropertyName == "Visibility")
-                SearchAndUpdateResults();
```

Comment:
We do not need call search for second time.
This event has been raised, when node's visibility was changed.
Node's visibility changed, when custom workspace was opened. (Visibility of Input and Output node).